build/Projects/OpenTelemetry.Instrumentation.ConfluentKafka.proj

```diff
@@ -0,0 +1,33 @@
+<Project>
+
+  <PropertyGroup>
+    <RepoRoot>$([System.IO.Directory]::GetParent($(MSBuildThisFileDirectory)).Parent.Parent.FullName)</RepoRoot>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <SolutionProjects Include="$(RepoRoot)\src\OpenTelemetry.Instrumentation.ConfluentKafka\OpenTelemetry.Instrumentation.ConfluentKafka.csproj" />
+    <SolutionProjects Include="$(RepoRoot)\test\OpenTelemetry.Instrumentation.ConfluentKafka.Tests\OpenTelemetry.Instrumentation.ConfluentKafka.Tests.csproj" />
+    <SolutionProjects Include="$(RepoRoot)\examples\redis\**\*.csproj" />
```

Comment:
I think that you have wrong folder here.
```suggestion
    <SolutionProjects Include="$(RepoRoot)\examples\Examples.ConfluentKafka\Examples.ConfluentKafka.csproj" />
```

Reply:
good catch thanks!